drake/systems/plants/test/CMakeLists.txt

```diff
@@ -88,6 +88,10 @@ if (LCM_FOUND)
     add_executable(compareRigidBodySystems compareRigidBodySystems.cpp)
     target_link_libraries(compareRigidBodySystems drakeRBSystem ${GTEST_BOTH_LIBRARIES})
     add_test(NAME compareAcrobotURDFtoSDF COMMAND compareRigidBodySystems Acrobot.urdf Acrobot.sdf WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/examples/Acrobot")
+
+    if (WITH_SNOPT)
```

Comment:
this should be snopt_c_FOUND, I believe.  (there are many other instances in the cmake files)